src/components/RadioButton/docs/RadioButtonTestStory.tsx

```diff
@@ -29,6 +30,19 @@ function renderTest(props?: any) {
           data-hook="radio-button-default"
         />
       </div>
+      <div style={{ marginTop: '50px' }}>
+        {TPAComponentsWrapper({ mobile: true })(
+          <RadioButton
+            value={'Checked'}
+            checked
+            withFocusRing
+            onChange={(val) => console.log(val)}
+            theme={RadioButtonTheme.Default}
+            label="default theme"
```

Comment:
Maybe it would be better to use a more descriptive label here:
```suggestion
            label="no focus-ring on mobile"
```